linkd-lib/src/logging.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright © 2021 The Radicle Link Contributors
```

Comment:
I was thinking, logging might be useful to the `rad` CLI as well, so wondering if we'd be interested in putting this in `rad-clib`?

Reply:
Generally yes, but we should build a proper :diamond_shape_with_a_dot_inside: and move the shared code (signer things, logging, etc.) into its own crate, otherwise linkd would depend on rad, which strikes me as out.